tool/tsh/db.go

```diff
@@ -659,12 +660,15 @@ func onDatabaseConnect(cf *CLIConf) error {
 		return trace.Wrap(err)
 	}
 	log.Debug(cmd.String())
+
+	peakStderr := utils.NewCaptureNBytesWriter(100)
```

Comment:
Why not capture the entire stderr? It might be hard to guess which number of bytes we need to place here if we want to handle other database command errors.

Reply:
Yeah, I haven't considered this. In this case, limiting the output captured seems necessary. I'm still not sure about the size, but making it a constant might be easier to extend later (if needed).